test/test-applications/integrations/dependency-libs/TestApplication.Shared/ConsoleHelper.cs

```diff
@@ -23,8 +26,12 @@ public static class ConsoleHelper
     public static void WriteSplashScreen(string[] args)
     {
         Console.WriteLine($"Command line: {string.Join(" ", args)}");
-        var (isProfilerAttached, additionalMessage) = ProfilerHelper.IsProfilerAttached();
```

Comment:
When I was working on otel-operator this was really helpful.
Do you see any option to put that information in the logs?
At the end of application lifecycle?

Reply:
What information was useful? That OpenTelemetry.AutoInstrumentation was loaded or that the CLR Profiler was attached to the process?

For the former we could list the loaded assemblies and check for specific assemblies. For the second there is already a native API, but don't use the wrapper from OpenTelemetry.AutoInstrumentation (otherwise that will end up loading the assembly again).